web-app/CKS.Web/Startup.cs

```diff
@@ -51,16 +47,40 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             }
 
 			app.UseDefaultFiles();
-			app.UseStaticFiles();
+			app.UseStaticFiles(new StaticFileOptions
+			{
+				OnPrepareResponse = ctx =>
+				{					
+					var fileName = ctx.File.Name;
+					var fileExtension = Path.GetExtension(fileName).ToLower();
+					var durationInSeconds = 0;
+					var cacheHeaderValues = new List<string> { "public" };
+
+					if (fileExtension == ".css"
```

Comment:
I think this list of values overcomplicates things. Now we only have 2 conditions there's no need although I can see why it was there when there multiple if statements. I think it can be as simple as:

```csharp
var headers = ctx.Context.Response.Headers;
var fileName = ctx.File.Name.ToLower();
if(fileName == "sw.js" || fileName.Contains(".html")) {
  headers[HeaderNames.CacheControl] = "public,must-revalidate,max-age=0";
}
else if(fileName.EndsWith(".css") || fileName.EndsWith(".js") || ctx.Context.Request.Path.Value.StartsWith("/static/")) {
  headers[HeaderNames.CacheControl] = "public,immutable,max-age=31536000";
}
```

Reply:
This is nice code. I had to tweak a little for a failing test though.